lib/CommandCall.js

```diff
@@ -25,11 +25,17 @@ const {
   iXmlNodeShClose,
 } = require('./ixml');
 
-const availableCommands = ['sh', 'cl', 'qsh'];
+const availableCommands = ['sh', 'cmd', 'qsh'];
+
 class CommandCall {
+  /**
+   * @description Creates a new CommandCall object
+   * @constructor
+   * @param {object} config
+   */
   constructor(config) {
     if (!config.command || !config.type) {
-      throw new Error('Please specify a command and type (cl, qsh, sh)');
+      throw new Error('Please specify a command and type (cmd, qsh, sh)');
```

Comment:
I preferred this is "cl", since it's more explicit. Could map to

Reply:
Agreed `cl` is clear compared to `cmd`.

Reply:
I see you reverted, but isn't the XMLService tag <cmd>? I think `cl` makes more sense, and not advocating reverting it back again, but just something to consider.

Reply:
The xml service tag is `<cmd>`